<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="MSI" ignored="0" name="LoginTest tests" tests="3" failures="1" timestamp="2020-03-24T22:10:17 EDT" time="27.565" errors="0">
  <testcase name="loginPOMPattern" time="0.558" classname="io.duotech.tests.LoginTest">
    <failure type="java.lang.AssertionError" message="did not expect to find [true] but found [false]">
      <![CDATA[java.lang.AssertionError: did not expect to find [true] but found [false]
at io.duotech.tests.LoginTest.loginPOMPattern(LoginTest.java:39)
at java.util.ArrayList.forEach(Unknown Source)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- loginPOMPattern -->
  <testcase name="loginPOMPatternNegative" time="0.317" classname="io.duotech.tests.LoginTest"/>
  <testcase name="testRun" time="0.002" classname="io.duotech.tests.LoginTest"/>
</testsuite> <!-- LoginTest tests -->
